### ISSUE TITLE
Report misuse of TypeVar if it's used only once in a generic method

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#1103]
The following should generate errors:
```python
def foo(a: _T) -> int:
    ...

def foo() -> _T:
    ...
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.80, which I just published. It will also be in the next release of pylance.
